generator_hdl.py

```diff
@@ -135,10 +138,17 @@ def writeTopModule_interface(topmodule_name, process_list, memDict, memInfoDict,
             if memInfo.isFIFO:
                 string_output_mems += writeTrackStreamRHSPorts_interface(mtypeB, memDict)
             else:
-                string_output_mems += writeMemoryRHSPorts_interface(mtypeB, memInfo)
+                  if not (("TPROJ" in mtypeB or "VMSME" in mtypeB) and args.split):
+                    string_output_mems += writeMemoryRHSPorts_interface(mtypeB, memInfo,memDict)
```

Comment:
@jasonfan393 Please decrease the indent to match the style elsewhere:
```suggestion
                if not (("TPROJ" in mtypeB or "VMSME" in mtypeB) and args.split):
                  string_output_mems += writeMemoryRHSPorts_interface(mtypeB, memInfo,memDict)
```